src/Social/Models/ChannelMessages.php

```diff
@@ -17,4 +17,40 @@ public function initialize()
         
         $this->setSource('channel_messages');
     }
+
+    /**
+    * Return all the messages of a channel
+    *
+    * @param Channels $channel
+    * @param array $filter
+    *
+    * default $filter = ["page" => 1, "limit" => 25, "order_by" => "id", "sort" => "DESC"]
+    *
+    * @return Simple
+    */
+    public function getMessages(Channels $channel, int $page = 1, int $limit = 25, string $orderBy = "id", string $sort = "DESC"): Simple
+    {
+        $appData = Di::getDefault()->get('app');
+        $userData = Di::getDefault()->get('userData');
```

Comment:
@rwhite27 use this instead of that https://github.com/bakaphp/baka/blob/0.6/src/Auth/UserProvider.php#L32